src/main/java/org/gridsuite/shortcircuit/server/repositories/specifications/FeederResultSpecificationBuilder.java

```diff
@@ -5,29 +5,27 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-package org.gridsuite.shortcircuit.server.utils;
+package org.gridsuite.shortcircuit.server.repositories.specifications;
 
 import org.gridsuite.shortcircuit.server.dto.ResourceFilter;
 import org.gridsuite.shortcircuit.server.entities.FeederResultEntity;
 import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.UUID;
 
 /**
  * @author Florent MILLOT <florent.millot@rte-france.com>
  */
-public final class FeederResultSpecificationBuilder {
+@Service
+public class FeederResultSpecificationBuilder {
```

Comment:
Why did you change it ? It's still static

Reply:
I could have kept it static but I prefer to keep both microservices as close as possible. After all, they do almost the same things, so making them different will only be confusing.

Reply:
I could have kept it static but I prefer to keep both microservices as close as possible. After all, they do almost the same things, so making them different will only be confusing.

I see no reason not to leave it 'final' though.